_posts/2019-11-18-experience-agile-2019-part-2.md

```diff
@@ -0,0 +1,221 @@
+
+#### Releasing the idea
+In my example, this stage happens in the demo of the new functionality for the manual calculations at the end of the sprints.  
+The manual calculations can be tested in the acceptance environment by the product owners and the final users.
+
+> An enterprise idea pipeline provides transparency of your options and allows you to quickly be aware of and respond to high-value work  
+> M. Moreira
+
+
+## Agile Fluency Project - Why focusing teams rock, by Diana Larsen - written by Michaela Broeckx
+
+Diana Larsen, or @DianaOfPortland, was an honourable guest at eXperience Agile conference. 
+Browsing her website FutureWorks Consulting, and googling her name, it becomes clear she had an amazing journey leading up to this *Agile Fluency Project*, throughout which you can move the *Agile Fluency Model* from idea to implementation, a model she co-created with James Shore. 
+
+The concept of *fluency* is omnipresent in most of her talks and articles, and related to that *team collaboration* has been one of her primary topics throughout her career. 
+
+### What is fluency?
+
+Fluency is routine practice mastery that persists under stress. In Lean, we would call this *kata*.
+You could say it is the skillful ease that comes from investing in learning. 
+
+That investment comes down to taking the time and making the effort for deliberate practice. 
+By regularly and consistently practicing a skill with increasing levels of challenge and with the intention of mastering that skill, a state of fluency can be reached. 
+The key is to not give up easily, and with that intent, to keep up your practice until it becomes a second nature... a bit like practicing your cycling skills as a kid, because you long to get rid of those silly training wheels.  
+
+### From team to organisation
+The Agile Fluency Model fist focuses on *team fluency*, a form of fluency that transcends the individual practice, and just like a team is more than the sum of its parts, team fluency also depends on management structures, relationships, and organizational culture, as well as the tools, technologies, and practices the teams use. 
+Team fluency is what you get when highly performant teams become unconsciously competent at collaborating and co-creating. 
+Fluency is the outcome of investment in learning and deliberate practice, and for team fluency this means learning together as a team. 
+
+There are a few stages before your team gets to fluency. 
+You need to invest in mastering the agile fundamentals to help focus as a team and you need a shift in team skills to be able to build sustainability while delivering value. 
+The Fluency model helps you discover the topics you can explore and practice to improve these aspects and make these shifts. 
+
+From there onwards the picture becomes broader, and from teams we move to the organisational structure and culture. 
```

Comment:
From there onwards, the picture becomes broader and from teams we move to the organisational structure and culture.